client/device/ios/create.go

```diff
@@ -121,7 +121,18 @@ func Create(ctx context.Context, client http.Client, createInp CreateInput) (*Cr
 		publicKey = &connectorReadRes.PublicKey
 	}
 
-	err = retry.Do(ctx, iosconfig.UntilState(ctx, client, deviceCreateOutp.Uid, state.PRE_READ_METADATA), *retry.NewOptionsWithLogger(client.Logger))
+	err = retry.Do(
+		ctx,
+		iosconfig.UntilState(ctx, client, deviceCreateOutp.Uid, state.PRE_READ_METADATA),
+		retry.NewOptionsBuilder().
+			Message("waiting for IOS device creation to be done").
```

Comment:
Waiting for IOS device to be onboarded to CDO...

Reply:
done 👍